### ISSUE TITLE
ImportError in sounds module

### ISSUE THREAD

**Issue by jimpanseiscool** [ev3dev/ev3dev-lang-python-demo#37]
I have commented on the commit https://github.com/ev3dev/ev3dev-lang-python-demo/commit/a5a84a3a9205090fcb976666291ce08b5ec9410b#diff-d6aca360016ad06f1a0286e0d04f2440 that latest change on the sounds module broke the demo.

The old version of the sounds module still works  on my ev3dev stable box.

**Comment by WasabiFan** [comment]
The master branch has in-development code which is only expected to be compatible with the most recent development version of the main library, running on stretch. If that isn't you, use ev3dev-jessie with the `jessie` branch of this repo.

**Comment by bittner** [comment]
@WasabiFan This is a valid point, though it causes quite some pain. Is there an easy way to offer a more recent build (daily or weekly) on [ev3dev.org/docs/getting-started](http://www.ev3dev.org/docs/getting-started/)? I would find it logical if the canonical source for installing ev3dev and the canonical source for demos of the very same would somehow match. It should be easy to get the demos running.

**Comment by WasabiFan** [comment]
A more recent build of what? The only thing made available on that page is a link to an ev3dev-jessie download, which is the currently-recommended ev3dev version.

**Comment by bittner** [comment]
... which doesn't match the demos in this repository.

If we want to support newcomers we ought to make sure there are no obstacles in the way to make demos run. When demos run only on stretch it should be easy to install ev3dev based on stretch. If ev3dev based on jessie is still recommended the demos that run only on stretch should be in a development or feature branch. Simple as that.

**Comment by WasabiFan** [comment]
The demos for Jessie are in the [_jessie_ branch](https://github.com/ev3dev/ev3dev-lang-python-demo/tree/jessie). This repository was established after the switch to focusing on Stretch so it has never had content or explanations written for Jessie. Is your complaint that _jessie_ isn't the default branch? That's not something I can change, but perhaps @dwalton76 can.

**Comment by bittner** [comment]
Exactly.

I think it should be the other way round: `master` should have Jessie and a `stretch` branch should have the new features for the upcoming stable ev3dev image, which is not yet recommended for beginners.

This would avoid issues like the ones with were faced with today.

**Comment by WasabiFan** [comment]
If there's a wish to change branch names, we should follow the convention used elsewhere in ev3dev: have `jessie` and `stretch` branches without an ambiguous `master`. The solution to your problem, however, is to change the default branch to match the current primary ev3dev version.

**Comment by bittner** [comment]
Then let's do it! Rename `master` to `stretch` and make `jessie` the default branch.

Note that this is not _my_ problem, but I see that this is a pitfall for newcomers. There should be no guessing why things don't "just work" out of the box.

**Comment by dwalton76** [comment]
I can rename branches this weekend

**Comment by dwalton76** [comment]
ok so we have

**ev3dev**
https://github.com/ev3dev/ev3dev
- The only branches are ev3dev-jessie (the default) and ev3dev-stretch

**ev3dev-lang-python**
https://github.com/ev3dev/ev3dev-lang-python/branches
- more branches than I care to type out here, many are stale.  @WasabiFan @ddemidov are any of these we can delete?
- `develop` is the default branch which is designed to work with stretch
- there are also `master` and `jessie` branches

**ev3dev-lang-python-demo**
https://github.com/ev3dev/ev3dev-lang-python-demo
- the `master` branch is the default, it is designed to work with stretch
- there is a also a `jessie` branch

Agreed this is really confusing :-1: Proposal of things we should do
- ev3dev-lang-python delete as many stale repositories as we can and see where that leaves us
- ev3dev-lang-python rename `develop` to `stretch`
- ev3dev-lang-python can we delete `master` since we already have a `jessie` branch?
- ev3dev-lang-python-demo rename `master` to `stretch`
- Make all three of these repos agree on their default (jessie vs stretch). ev3dev defaults to jessie while ev3dev-lang-python and ev3dev-lang-python-demo default to stretch. @dlech what are your thoughts on this, are you close to switching the default in ev3dev to stretch?

**Comment by WasabiFan** [comment]
> more branches than I care to type out here, many are stale. @WasabiFan @ddemidov are any of these we can delete?

:+1: I just deleted a variety of branches that had been merged previously, so it should be a bit cleaner.

> Agreed this is really confusing 👎 Proposal of things we should do

I agree with that plan. I'm not entirely sure what the master branch in ev3dev-lang-python was for, but it points to the same ref as `jessie` right now anyway.

> Make all three of these repos agree on their default (jessie vs stretch).

I think we should default to jessie until it is no longer the primary recommended platform.

**Comment by dwalton76** [comment]
Defaulting everything to jessie sounds good to me

**Comment by dwalton76** [comment]
@WasabiFan @ddemidov @gregcowell @bittner @dlech FYI made the following branch changes to ev3dev-lang-python-demo:
- default is now `jessie`
- renamed `master` to `stretch`

You will need to jump through some hoops to get the branches of your fork of ev3dev-lang-python-demo to match exactly.  If you do not have any outstanding work in your fork, do not have any stars, etc the easiest thing to do is just delete your fork and re-fork ev3dev-lang-python-demo.

If you do have stars, etc you'll have to google around a bit to find the git commands to make your branches match with the parent repo....I don't now what they are off the top of my head.  I did not have any stars so I just deleted my repo and re-forked.
